### DISCUSSION TITLE
Stand up Thursday May 18th

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [Bugs-io/TC3001B#79] [Main Stand Ups]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
- 

**What are your priorities for the next 24 hours?**:
- 

**Blockers**:
- 

**Shoutouts**:
- @username for x
```

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
- Worked on graphs with carbon-charts
- Worked on testing with PonyORM

**What are your priorities for the next 24 hours?**:
- Reach out to @Jacobgtz024 to organize the development of the graphs
- Worked on Luis Gerardo's deliverable

**Blockers**:
- None

**Shoutouts**:
- @BrandonMagana for doing yesterdays activity

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
-  Worked on web .csv CRUD  activity. Implemented 2 endpoints (upload and download)

**What are your priorities for the next 24 hours?**:
-  Work on Luis Gera's architecture document 

**Blockers**:
-  Adan's class

**Shoutouts**:
- todas for being in class yesterday

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
- Introduced linter on front and fixed a ton of errors. ACCEPT: https://github.com/Bugs-io/ibm-dashboard-app/pull/18
- Create a grid that is drag, drop and sorts itself after.
- Write down "estrategia de solución" on arch document.

**What are your priorities for the next 24 hours?**:
- Polish the grid and even introduce test charts
- Add the lock functionality to the grid

**Blockers**:
- None

**Shoutouts**:
- @Jacobgtz024 for interviewing again with Oracle!
- @BrandonMagana for doingthe web act.
